### PR TITLE
Fixed RTL issue for "Mobile Add-ons site" message banner

### PR DESCRIPTION
**Pull request by ahmedhasan07**
#4791 issue is fixed. 

Before and after screenshot is attached. 

Before
screen shot 2017-06-17 at 20 05 11
After
screen shot 2017-06-17 at 20 05 00

### CODE REVIEW

**Comment by ahmedhasan07** [comment]
@atiqueahmedziad for helping :smile:

**Comment by caitmuenster** [comment]
Thanks for the patch, @ahmedhasan07! Your contribution has been added to our [recognition wiki ](https://wiki.mozilla.org/Add-ons/Contribute/Recognition#June_2017) and your Mozillians profile has been vouched. :) 

Welcome onboard! I look forward to seeing you around!

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -566,6 +566,10 @@ body:not(.home) .amo-header,
   margin-top: 22px;
   padding: 15px 45px 15px 15px;
 
+  .html-rtl & {
```
Comment (atiqueahmedziad) [inline static/css/restyle/restyle.less]: There is a small typo/mistake. It should be `.html-rtl&` instead of `.html-rtl &`

Comment (eviljeff) [inline static/css/restyle/restyle.less]: can you create a pull request?

Comment (atiqueahmedziad) [inline static/css/restyle/restyle.less]: Sure. Here is the pr- https://github.com/mozilla/addons-server/pull/5740

**Pull request merged**